server/middlewares/errorHandler.js

```diff
@@ -0,0 +1,40 @@
+"use strict"
+
+function errorHandler(err, req, res, next) {
+    if(err.name === "SequelizeValidationError") {
+        const errMessages = []
```

Comment:
[OPTIONAL] Kalau ada waktu, nanti coba baca2 tentang es6 built in lain, seperti map, filter, reduce, dll.. https://medium.com/poka-techblog/simplify-your-javascript-use-map-reduce-and-filter-bd02c593cc2d